.github/workflows/static-analysis.yml

```
@@ -1,13 +1,46 @@
+        run: composer sa
+
+      - name: Run PHP Mess Detector
+        run: ./vendor/bin/phpmd src text ./phpmd.xml
+
+      - name: Download ComposerRequireChecker PHAR
+        run: |
+          curl -L https://github.com/maglnet/ComposerRequireChecker/releases/latest/download/composer-require-checker.phar -o composer-require-checker.phar
+          chmod +x composer-require-checker.phar
+
+      - name: Run composer-require-checker without config
+        if: ${{ !inputs.has_crc_config }}
```

actionlint (1.7.7)

property "has_crc_config" is not defined in object type {}

(expression)